plugfest/index.html

```diff
@@ -1,239 +1,274 @@
+      list-style: none;
+      padding: 15px 0;
+    }
+
+    footer li,
+    footer a {
+      display: inline-block;
+      color: white;
+      font-family: 'Open Sans', sans-serif;
+      font-size: 17px;
+      padding: 0 5px;
+      text-decoration-line: none;
+    }
+
+    footer p {
+      color: white;
+      font-family: 'Open Sans', sans-serif;
+      font-size: 14px;
+      padding-top: 12px;
+    }
+
+    form {
+      border: solid 1px black;
+      margin-top: 5px;
+      margin-bottom: 5px;
+      padding-left: 10px;
+      padding-right: 10px;
+    }
+
+    .form-control {
+      width: 100%;
+      text-align: right;
+    }
+
+    table {
+      width: 100%;
+      text-align: center;
+    }
+
+    textarea {
+      resize: vertical;
+      width: calc(100% - 2*3px);
+    }
+
+    .btn {
+      background: rgb(51, 184, 164);
+      border: 1px solid transparent;
+      border-radius: 200px;
+      color: white;
+      padding: 12px 25px;
+      font-size: 16px;
+      cursor: pointer;
+    }
+
+    .highlighted {
+      text-align: center;
+      font-style: italic;
+    }
+
+    .framed {
+      padding: 12px 25px;
+      font-size: 16px;
+    }
+
+    .framed a {
+      color: white;
+    }
+
+    p.success {
+      background: rgb(51, 184, 164);
+      color: white;
+    }
+
+    p.failure {
+      background: rgb(161, 33, 33);
+      color: white;
+    }
+
+    table.success tr {
+      background: rgb(87, 109, 132);
+      color: white;
+    }
+
+    table.failure tr {
+      background: rgb(161, 33, 33);
+      color: white;
+    }
+  </style>
+</head>
+
+<body>
+  <header>
+    <div class="container">
+      <img alt="Thingweb logo" src="./img/logo.png">
+    </div>
+  </header>
+
+  <div class="wrapper">
+
+    <h1>Eclipse Thingweb Services for W3C WoT</h1>
+
+    <p>
+      Eclipse Thingweb provides services to support Web of Things developers.
+      These services are listed below.
+    </p>
+
+
+    <h2>Thing Description Playground</h2>
+
+    <p>
+      Playground is a tool to validate TDs and manipulate them for usage in other tools.
+      You can do validation to check conformance, generate Open API and Async API documents, add or remove default
+      values, visualize TDs and find spelling mistakes in TDs.
+    </p>
+
+    <ul>
+      <li><a href="./playground/">Online Tool</a></li>
+      <li><a href="https://github.com/thingweb/thingweb-playground">GitHub Repository</a></li>
+    </ul>
+
+
+    <h2>Web-based Interactions with Things</h2>
+
+    <p>
+      Using the tools below, you can interact with different WoT Things over your browser.
+    </p>
+
+    <ul>
+      <li>
+        <a href="http://plugfest.thingweb.io/webui/">Browserified node-wot</a></li>: You can try interacting with different Things using this simple interface. Note that this just a demo page to showcase the <a href="https://github.com/eclipse-thingweb/node-wot/tree/master/packages/browser-bundle">browser bundle of node-wot</a>.
```

Comment:
Different style of linking for webui .. plugfest before uses "./playground/" for the `href`

Reply:
done